site/content/recipes/perform_code_review.md

```diff
@@ -37,14 +37,35 @@
 
 ## Reviewer reviews code
 
-1. Reviewer reminds themselves of objectives and areas of focus from first meeting (link to Meet and Agree on Objectives section)
+What should you do if you're the reviewer, providing feedback on someone else's
+code?
```

Comment:
Reviewing code can be a difficult exercise. Where do you even start? What do you look at? The good news is: there are a number of points that are almost always worth looking at. Particularly, readability is often a good angle from which starting a code review. You can start simple, for instance looking at variables and functions' name: are they explicit? Do they carry intent? As you get more and more familiar with the code under review, move on to larger structure: can some of the functions be split into several functions? Do you see any complex conditional statements that you find hard to read? Eventually, perhaps after several iterations of the code review process, you may be able to criticise the code design, for instance its modules and the relationships between them. But don't start from there!

Reply:
As discussed in the Meet and Agree on objectives phase, however, code reviews are most effective when a clear goal is set. This goal can help you as a reviewer to orient your feedback, and focus on what matters from the point of view of this overall objective. It is also good practice to adjust your review to the author, particularly to their programing experience. For instance, if the code review aims at assessing readability in general, and you knew that the code author has limited experience, you would want to pay attention to variable and function names, parameter lists, complex conditionals, or lengthy functions. It would probably be best to keep away from suggesting using the latest and greatest features of the language. If the review is about writing more idiomatic code, however, the focus changes to looking specifically at how the code under review makes use of the language it is written in.

Reply:
I added these two paragraphs at the start of the section, with some minor revisions, and then wrote a transition into the paragraph about a "process with two elements"

Reply:
These paragraphs, while good, make this section quite a bit longer. Reading through the whole thing, it seems to jump around a bit and lacks some focus. I don't think this is a blocker, and it is something we can refine over time.